content/en/dashboards/widgets/wildcard.md

```diff
@@ -41,6 +42,22 @@ However, if none of the Datadog widgets meets your visualization needs, a Wildca
 1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
 1. Click **Save**.
 
+#### Formulas Scalar vs. Formulas Timeseries
+
+In Datadog dashboards, scalar and timeseries visualizations serve different purposes and are used to represent data in distinct ways.
```

Comment:
`Visualization` might not be the noun we're looking for, we  are referring to a data format (which has a corresponding "request type". Maybe something like

> visualizations are powered by a multiple _request types_, including scalar and timeseries. Each _request types_ changes the number and type of fields available for the data in a Wildcard Widget.

There are 2 other request types in the widget selector which are GA, List and Histogram. It's OK to not explain what all 4 of them in 1 PR, but we don't want people to have the impression that there are only 2 request types.